css/style.css

```diff
@@ -56,6 +56,90 @@ a.anchor{
 	}
 }
 
+/*Down arrow animation*/
+.arrows {
+	width: 60px;
+	height: 72px;
+	left: 50%;
+	position: relative;
+	margin-left: -30px;
+	margin-right: auto;
+	margin-top: 50px;
+	bottom: 20px;
+}
+
+.arrows path {
+	stroke: #fff;
+	fill: transparent;
+	stroke-width: 1px;	
+	animation: arrow 2s infinite;
+	-webkit-animation: arrow 2s infinite; 
+}
+
+.arrows-gdpr {
+	width: 60px;
+	height: 72px;
+	position:relative;
+	left: 50%;
+	margin-left: -30px;
+	margin-top: 10px;
+	bottom: 20px;
+}
+
+.arrows-gdpr path {
+	stroke: #7bb69d;
+	fill: transparent;
+	stroke-width: 1px;	
+	animation: arrow 2s infinite;
+	-webkit-animation: arrow 2s infinite; 
+}
+
+@keyframes arrow
+{
+0% {opacity:0}
+40% {opacity:1}
+80% {opacity:0}
+100% {opacity:0}
+}
+
+@-webkit-keyframes arrow /*Safari and Chrome*/
+{
+0% {opacity:0}
+40% {opacity:1}
+80% {opacity:0}
+100% {opacity:0}
+}
+
+.arrows path.a1 {
+	margin: auto;
+	animation-delay:-1s;
+	-webkit-animation-delay:-1s; /* Safari 和 Chrome */
+}
+
+.arrows path.a2 {
+	animation-delay:-0.5s;
+	-webkit-animation-delay:-0.5s; /* Safari 和 Chrome */
+}
+
+.arrows path.a3 {	
+	animation-delay:0s;
+	-webkit-animation-delay:0s; /* Safari 和 Chrome */
+}
+
+.arrows-gdpr path.a1 {
+	animation-delay:-1s;
+	-webkit-animation-delay:-1s; /* Safari 和 Chrome */
+}
+
+.arrows-gdpr path.a2 {
+	animation-delay:-0.5s;
+	-webkit-animation-delay:-0.5s; /* Safari 和 Chrome */
+}
+
+.arrows-gdpr path.a3 {	
+	animation-delay:0s;
+	-webkit-animation-delay:0s; /* Safari 和 Chrome */
+}

```

Comment:
If we add a class to position the arrow in the gdpr page and use the .arrows class we can remove all the properties related to .arrows-gdpr to avoid duplication